src/style/index.css

```diff
@@ -18,4 +18,11 @@
 
 mjx-container.MathJax[jax='CHTML'][display='true'] {
   display: inline-block !important;
+  overflow-x: auto;
+  overflow-y: hidden;
+  padding: 0.5em;
```

Comment:
Småpirk: Denne fører til at inline matteformler ikke ligger på samme linje som teksten ellers i dokumentet (se bildene under).
Kanskje vi heller skulle ha `padding: 0.5em 0.5em 0 0.5em;`
Old:
![image](https://user-images.githubusercontent.com/43534577/106254753-6568bf00-6219-11eb-98fb-963e66990a27.png)

New:
![image](https://user-images.githubusercontent.com/43534577/106254689-4d913b00-6219-11eb-868a-d00ca410df5a.png)

Reply:
Det hadde sett ganske rart ut da tror jeg.
Og det er derfor jeg har top variabelen. Problemet er at alt ettersom hva som er i formelen så har den forskjellig høyde, og da må top være en forskjellig verdi for å få sentrert den

Reply:
https://i.imgur.com/X9mjLt5.png
Her

Reply:
kunne man lagt til en margin-top: -0.5 kanskje?

Reply:
Jeg ser nå at det nok ikke er så lett som jeg antok, men jeg mener noe burde gjøres med. Slik det er nå ser det svært rotete ut med inline mathjax imo. Se [https://ndla-frontend-pr-622.ndla.sh/subject:32/topic:1:165872/topic:1:165911/resource:1:107582?filters=urn:filter:4200b774-5b7b-4900-bf1d-e0b298b9cb97](https://ndla-frontend-pr-622.ndla.sh/subject:32/topic:1:165872/topic:1:165911/resource:1:107582?filters=urn:filter:4200b774-5b7b-4900-bf1d-e0b298b9cb97)

Reply:
I eksempelet du sendte nå @hakmet så fikk jeg plassert alle elementer fint ved å legge til følgende:
```
vertical-align: bottom;
position: initial !important; // Kun av grunnen som nevnt undder.
```

Position er satt til relative direkte med style="position: relative" fra før av så det ut som og den måtte fjernes for at det skulle fungere.

Reply:
Med den der så blir det som var off litt bedre, men for meg ser det ut som at det som sto på linja fra før av blir dytta lengre ned og derfor av linja. (Om det gir mening. Kan lasta opp screenshot av det jeg mener om ønskelig)

Reply:
https://i.imgur.com/4sAlfNj.png
Får fortsatt slik da, men er kansje bedre

Reply:
Alternativt prøv:

```
vertical-align: middle;
margin: 5px 0; // Tror dette er nødvendig om vertical-align skal brukes.
position: initial !important;
```

Reply:
Ja, ser ut som den egentlig blir litt for lav da.

Reply:
Jeg så på hvordan lignende ting var gjort i for eksempel stackoverflow, og det ser ut som om kodeformaterte snutter med bakgrunnsfarge er midtstilt horisontalt. Så jeg tror det er den beste løsningen